### ISSUE TITLE
current pathfinding is good, but the previous was better.

### ISSUE THREAD

**Issue by Hellsice** [awgil/ffxiv_vfallguy#5]
Although it is nice to see the pathfinding jump around less, the updates were very useful midrun.
The current version chooses one path and only adjusts after you've passed a "checkpoint line". And the adjustments sometimes sends you straight into an aoe. It once sent me in a straight line through the square aoe's from the bottom straight to the top of it. You cannot dodge that. Another instance of bad pathing, if you decide to not follow the path, it will for a long time not update and just say you need to walk through different aoe's.
The path also has very strict timing to dodge the aoe's, and if you are delayed by even 0.05 seconds, the entire path created no longer works. Which happens quite often because the pathing also goes very tight around objects or goes straight through a stun block (center of a hammer). 
I think this in general would be fixed again if the timers that were on each checkpoint returned, to see if we are on pace with the laid out path. Or if you reroll the pathfinding update to the previous version and then combine it with the current one to have a less sporadic initial path.

tldr: current pathfinder needs the timers back, and more frequent updates to pathing midrun so it doesn't send you into an aoe.

rereading this it makes me feel like the current pathfinding has "checkpoints" spread around the map and whenever you pass one, it snapshots all aoe's between the passed checkpoint line and next checkpoint line. And uses the snapshot to determine the pathing.

i'll check the history to see if i can find something, but i'll likely not find anything as i don't know C#

**Comment by Hellsice** [comment]
bit more testing really shows that a good path is created for the first part of the stage, but with incredibly tight windows to dodge aoe's. One fail and the pathfinding breaks.
the second part of the stage (hammers) it really cannot do at all. It keeps sending me through the center of a hammer (which stuns you) or the path goes in such a way that you are guaranteed to be hit by a hammer. I have stopped using the pathfinding for that part

**Comment by yuyukim** [comment]
I'm having the same problem.

**Comment by nirustim** [comment]
yeah the more frequently updating pathfinding was good

**Comment by BSDGurl** [comment]
wow this has been an issue since 2023 ; . ; and you're saying it worked and they took away it working and still have not put it back that really just says this plugin is just a troll
